### DISCUSSION TITLE
Don't want to test this myself but just wanting to make sure I did this right

### DISCUSSION THREAD

**Discussion by voxvargr** [PartialVolume/shredos.x86_64#143] [Q&A] [answered]
long story short, a friend is going to be wiping his work PC before submitting his resignation and asked for a usb that will just do it, this is what I found and want to give him. when customizing the commands in the grub.cfg this is what I put in for the commands

`set default="0"
set timeout="0"

menuentry "shredos" {
	linux /boot/shredos console=tty3 loglevel=3	nwipe_options="--autonuke --autopoweroff --nosignals -nogui -nousb
}`

just want to confirm this will automatically nuke everything without user input and won't erase the usb that's plugged in, is this correct?

**Comment by PartialVolume** [comment, marked as answer]
The syntax is not quite correct, you're missing a couple of hyphens and the closing quote, once those are fixed it would work.

Using those options you would almost get a silent autonuke, however if there is a CDROM it would display a read-only warning but will still wipe any discs that are present. You would need to add `--exclude=/dev/sr0` if you wanted a perfectly blank screen.

In --nogui mode nothing gets printed to the main terminal. So you have just a blank screen, the only way you know anything is happening is from the disc activity LED. I don't remember if it prints any progress after a while but it's possible it's perfectly silent throughout the entire duration of the wipe.

Correct syntax with exclusion of the CD/DVD drive.
```
	linux /boot/shredos console=tty3 loglevel=3     nwipe_options="--autonuke --autopoweroff --nosignals --nogui --nousb --exclude=/dev/sr0"
```

However, you don't need to hide the GUI to do a autonuke. My preference is to display the GUI so you can see progress.

autonuke with GUI (no user interaction required, will immediately autonuke without user interaction required to initiate the wipe)
```
	linux /boot/shredos console=tty3 loglevel=3     nwipe_options="--autonuke --autopoweroff --nousb"
```

With reference to --nogui mode , if you wanted to see progress, you can send nwipe a signal `kill -10 [nwipe's_processID]` and it will log the progress to the nwipe log file, so if you wanted to see progress you could switch to ALT-F3 and then you can setup a constant monitoring of the log file with `tail -f nwipe{what_ever_the_log_files_called}.txt` then from ALT-F2 issue `kill -10 [nwipe's_processID]`

**Reply by voxvargr** [reply]
`set default="0"
set timeout="0"

menuentry "shredos" {
	linux /boot/shredos console=tty3 loglevel=3	nwipe_options="--autonuke --autopoweroff --nousb"
 
}`

Thanks a ton!

I think I got it correct now. I don't need a blank screen, just don't want to overwhelm my buddy when he uses it lol

**Reply by PartialVolume** [reply]
No problem :-)

**Reply by PartialVolume** [reply]
One other thing to remember is if the laptop is new and has legacy support disabled, or not available and secure boot is enabled, ShredOS won't boot. He would need to disable secure boot in the bios first. Other than that, it should work.
